manifests/vsphere-67u3/vanilla/deploy/vsphere-csi-node-ds.yaml

```diff
@@ -0,0 +1,125 @@
+kind: DaemonSet
```

Comment:
Can we add a comment in the beginning with the min k8s version?  Now that this is under vsphere-67u3, it is not clear what k8s version is supported.

Reply:
added following comment
```
# Minimum Kubernetes version - 1.16
# For prior releases make sure to add required --feature-gates flags for side car containers
```